src/utils/get-events-date.ts

```diff
@@ -0,0 +1,23 @@
+import { Event } from '@/data/events';
+
+// Convert event date and return upcoming events
+export function getUpcomingEvents(EVENTS: Event[]): Event[] {
+    const currentDate = new Date();
+    return EVENTS.filter((event) => {
+        const eventDate = new Date(
+            `${event.date.year} ${event.date.month} ${event.date.day} ${event.date.endTime}`
+        );
+        return eventDate >= currentDate;
+    });
+}
+
+// Convert event date and return past events in reversed order
```

Comment:
By "reversed order" do you mean the most recent event is at the top? If so, maybe make this clearer in the comment